### PR TITLE
Update README and move large sections under `docs/`

### PR DESCRIPTION
**Pull request by aaronbuchwald**
This PR updates the HyperSDK README:

- move the features section into `./docs/explanation/Features.md`
- moves glossary of terms under `./docs/reference/Glossary.md`
- updates config options to be stored under the name `services`
- Adds a short how-to guide on configuring an option defined service via config/programmatically
- Simplifies the HyperSDK and MorpheusVM READMEs

### CODE REVIEW

File: docs/explanation/Features.md
Comment (joshua-kim) [inline docs/explanation/Features.md:1]: I believe its idiomatic to have lowercase markdown file names with exceptions for specific reserved files like README

File: tests/integration/integration.go
Comment (joshua-kim) [inline tests/integration/integration.go:1]: Should the non markdown code be in its own pr?

File: docs/explanation/Features.md
Comment (aaronbuchwald) [reply docs/explanation/Features.md:1]: Will update

File: tests/integration/integration.go
Comment (aaronbuchwald) [reply tests/integration/integration.go:1]: I renamed things, so that they made more sense as I was writing the documentation, but will separate this into its own PR.

File: docs/explanation/Features.md
Comment (aaronbuchwald) [reply docs/explanation/Features.md:1]: Updated
